### PR TITLE
fix #59682 egetelgraf.com

### LINKED ISSUE #59682: egetelgraf.com
### Issue URL (Ads)
 [https://www.egetelgraf.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.egetelgraf.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (2880), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#59682

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -1705,7 +1706,7 @@ paylasaninsanlar.blogspot.com##.sidebar > #HTML2
 paylasaninsanlar.blogspot.com##.sidebar > #HTML4
 bianet.org##.banner_main
 bianet.org##.ad52-container
-thegeyik.com,shallwe.com.tr,paragezegeni.com,akillitelefon.com##.adace-slot-wrapper
+egetelgraf.com,thegeyik.com,shallwe.com.tr,paragezegeni.com,akillitelefon.com##.adace-slot-wrapper
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:1709]: it will break this site
image

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -68,6 +68,7 @@ macizlee1.com,macizlee.ne,dunyaatlasi.com,ensonhaber.com,gayrimenkulhaber.com,in
 gazeteulus.com##.reklammain
 golcukvizyongazetesi.com##.header-promo
 golcukvizyongazetesi.com##div[class$="reklamalanı"]
+egetelgraf.com##.g1-injected-unit
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:71]: cannot find

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply TurkishFilter/sections/specific.txt:71]: <details>
<summary>element</summary>

</details>

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply TurkishFilter/sections/specific.txt]: I don't know why they put this video in ad element.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**